security-policies/bundle/compliance/cis_azure/rules/cis_3_11/rule.rego

```diff
@@ -0,0 +1,30 @@
+package compliance.cis_azure.rules.cis_3_11
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import future.keywords.if
+
+finding = result if {
+	# filter
+	data_adapter.is_storage_account
+	_ = data_adapter.resource.extension.blobService
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(ensure_soft_delete_is_enabled),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default ensure_soft_delete_is_enabled = false
+
+ensure_soft_delete_is_enabled if {
```

Comment:
```suggestion
soft_delete_is_enabled if {
```

nit: `ensure` implies taking an action, not just checking

Reply:
Done